File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Reflection;
using DevExpress.Utils.Drawing;
using DevExpress.XtraEditors.Drawing;
using DevExpress.XtraEditors.Controls;
using DevExpress.LookAndFeel;

namespace WindowsApplication18 {
    public partial class Form1 : Form {
        public Form1() {
            SetMyUltraFlatPainter();
            InitializeComponent();
        }

        void SetMyUltraFlatPainter() {
            Type type = typeof(DevExpress.LookAndFeel.LookAndFeelPainterHelper);
            FieldInfo fi = type.GetField("painters", BindingFlags.Static | BindingFlags.NonPublic);
            BaseLookAndFeelPainters[] painters = (BaseLookAndFeelPainters[])fi.GetValue(null);
            painters[(int)ActiveLookAndFeelStyle.UltraFlat] = new MyUltraFlatLookAndFeelPainters(null);
        }
    } 
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports System.Reflection
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.LookAndFeel

Namespace WindowsApplication18
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            SetMyUltraFlatPainter()
            InitializeComponent()
        End Sub

        Private Sub SetMyUltraFlatPainter()
            Dim type As Type = GetType(DevExpress.LookAndFeel.LookAndFeelPainterHelper)
            Dim fi As FieldInfo = type.GetField("painters", BindingFlags.Static Or BindingFlags.NonPublic)
            Dim painters() As BaseLookAndFeelPainters = DirectCast(fi.GetValue(Nothing), BaseLookAndFeelPainters())
            painters(CInt((ActiveLookAndFeelStyle.UltraFlat))) = New MyUltraFlatLookAndFeelPainters(Nothing)
        End Sub
    End Class
End Namespace